page/docs/guide/postman.md

```diff
@@ -50,6 +50,45 @@ constructor(MockStarknetMessaging mockStarknetMessaging_) public {
 }
 ```
 
+### Postman - l1 to l2 mock endpoint
```

Comment:
I would document the two new endpoints as we do in `page/docs/guice/mint-token.md`:
`## Name of the functionality`
Description
`<METHOD> /endpoint`
Request: (say if none)
`Request body`
Response: (say if none)
`Response body`

Reply:
ok I'll do it at the end

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/365/commits/fd7f3fbc4471a6a585599c5cf45b48cd70645430